### DISCUSSION TITLE
Is pgsc_calc supported on non-Mac arm computers?

### DISCUSSION THREAD

**Discussion by AWS-crafter** [PGScatalog/pgsc_calc#347] [Q&A] [answered]
I have seen this:
https://pgsc-calc.readthedocs.io/en/latest/how-to/arm.html. Is pgsc_calc is also supported on arm Linux? If so, would I have to use 32GB instead of 16GB RAM, or would pgsc_calc be slower?

I am wondering because I would like to use the AWS Graviton.

**Comment by nebfield** [comment, marked as answer]
It should be OK. [We publish `linux/amd64` and `linux/arm64` container images](https://github.com/PGScatalog/pygscatalog/pkgs/container/pygscatalog/250193082?tag=pgscatalog-utils-1.2.0) for all of our dependencies 🥳 

When Docker is running on macOS it's running a Linux VM in the background, which is why it's helpful to allocate extra RAM on macs.
